### DISCUSSION TITLE
Websocket: Parsing: delimiter not found

### DISCUSSION THREAD

**Discussion by antscribano** [Marekkon5/onetagger#248] [Q&A] [answered]
Hello everyone,
I'm experiencing an issue when trying to tag tracks downloaded from Beatport.

Found this on the log:

> 2023-08-08 19:11:27 [WARN] onetagger_autotag: Error loading file: Missing artist tag!
> 2023-08-08 19:11:27 [INFO] onetagger_autotag: [spotify] State: Skipped, Accuracy: None, Path: [path]\Desktop\test\10899444_Tortured_Brain_Original_Mix.mp3
> 2023-08-08 19:11:27 [INFO] onetagger_autotag: Starting tagging: 1 files, 16 threads!
> 2023-08-08 19:11:27 [INFO] onetagger_autotag: Starting beatport
> 2023-08-08 19:11:27 [INFO] onetagger_autotag: Tagging: [path]\Desktop\test\10899444_Tortured_Brain_Original_Mix.mp3
> 2023-08-08 19:11:27 [WARN] onetagger_tag::id3: Failed loading: [path]\Desktop\test\10899444_Tortured_Brain_Original_Mix.mp3, creating new tag. Parsing: delimiter not found
> 2023-08-08 19:11:27 [WARN] onetagger_autotag: Error loading file: Missing artist tag!
> 2023-08-08 19:11:27 [INFO] onetagger_autotag: [beatport] State: Skipped, Accuracy: None, Path: [path]\Desktop\test\10899444_Tortured_Brain_Original_Mix.mp3
> 2023-08-08 19:11:27 [INFO] onetagger_autotag: Starting tagging: 1 files, 4 threads!
> 2023-08-08 19:11:27 [INFO] onetagger_autotag: Starting bandcamp
> 2023-08-08 19:11:27 [INFO] onetagger_autotag: Tagging: [path]\Desktop\test\10899444_Tortured_Brain_Original_Mix.mp3
> 2023-08-08 19:11:27 [WARN] onetagger_tag::id3: Failed loading: [path]\Desktop\test\10899444_Tortured_Brain_Original_Mix.mp3, creating new tag. Parsing: delimiter not found
> 2023-08-08 19:11:27 [WARN] onetagger_autotag: Error loading file: Missing artist tag!
> 2023-08-08 19:11:27 [INFO] onetagger_autotag: [bandcamp] State: Skipped, Accuracy: None, Path: [path]\Desktop\test\10899444_Tortured_Brain_Original_Mix.mp3
> 2023-08-08 19:11:27 [INFO] onetagger_autotag: Starting tagging: 1 files, 4 threads!
> 2023-08-08 19:11:27 [INFO] onetagger_autotag: Starting junodownload
> 2023-08-08 19:11:27 [INFO] onetagger_autotag: Tagging: [path]\Desktop\test\10899444_Tortured_Brain_Original_Mix.mp3
> 2023-08-08 19:11:27 [WARN] onetagger_tag::id3: Failed loading: [path]\Desktop\test\10899444_Tortured_Brain_Original_Mix.mp3, creating new tag. Parsing: delimiter not found
> 2023-08-08 19:11:27 [WARN] onetagger_autotag: Error loading file: Missing artist tag!
> 2023-08-08 19:11:27 [INFO] onetagger_autotag: [junodownload] State: Skipped, Accuracy: None, Path: [path]\Desktop\test\10899444_Tortured_Brain_Original_Mix.mp3
> 2023-08-08 19:11:27 [INFO] onetagger_autotag: Written failed songs to: [path]\AppData\Roaming\OneTagger\OneTagger\config\runs\failed-1691514687731.m3u, successful to: [path]\AppData\Roaming\OneTagger\OneTagger\config\runs\success-1691514687731.m3u
> 2023-08-08 19:11:27 [INFO] onetagger::socket: Tagging finished, took: 1 seconds.
> 2023-08-08 19:11:59 [WARN] onetagger_tag::id3: Failed loading: [path]\Desktop\test\10899444_Tortured_Brain_Original_Mix.mp3, creating new tag. Parsing: delimiter not found

So I tried to manually add the artist tag and got this error on the log:

> 2023-08-08 19:12:40 [ERROR] onetagger::socket: Websocket: Parsing: delimiter not found, Data: {"action":"tagEditorSave","changes":{"path":"[path]\\Desktop\\test\\10899444_Tortured_Brain_Original_Mix.mp3","changes":[{"type":"raw","tag":"TPE1","value":["Argon Sphere"]}],"separators":{"id3":", ","vorbis":null,"mp4":", "},"id3v24":false}}

Running OneTagger v1.7.0 on Win 11.

Any suggestion?
Thank you

**Comment by Marekkon5** [comment]
Hello, it seems like some of your tracks have incorrect ID3 frames, which 1T cannot read. Could you please send a track as an example? Thank you

**Reply by antscribano** [reply]
Here we go: https://drive.google.com/file/d/1RrHX29h7zWb2pUE-eenNqNWQVwbMmBwv/view?usp=sharing

Thanks

**Reply by Marekkon5** [reply, marked as answer]
It seems that your WWWAUDIOFILE and Publisher tags contain some non-text characters which cause 1t to not load the files properly. You can remove those using Kid3 or any other tag editor.
